src/types/ethr.rs

```diff
@@ -178,12 +178,14 @@ impl EthrBuilder {
     ///  When resolving DIDs with publicKey identifiers, if the controller (owner) address is different from the corresponding address of the publicKey, then the #controllerKey entry in the verificationMethod array MUST be omitted.
     ///
     ///  referecne: [spec](https://github.com/decentralized-identity/ethr-did-resolver/blob/master/doc/did-method-spec.md#controller-changes-didownerchanged)
-    pub fn owner_event(&mut self, event: DidownerChangedFilter) -> Result<()> {
+    pub fn owner_event(&mut self, event: DidownerChangedFilter) -> Result<bool> {
         self.controller(&event.owner)?;
+        let mut deactivated = false;
         if event.owner == Address::from_str(NULL_ADDRESS).expect("const address is correct") {
-            log::warn!("Address has been deactivated. not implemented");
+            // set the deactivated flag in case the address was deactivated.
+            deactivated = true;
         }
-        Ok(())
+        Ok(deactivated)
```

Comment:
I'm confused about this result.     Is deactivated successful or failure?

Reply:
it's neither. The deactivation is separate from the error.